fdbserver/ClusterController.actor.cpp

```diff
@@ -522,7 +522,7 @@ void checkBetterSingletons(ClusterControllerData* self) {
 
 	WorkerDetails newEKPWorker;
 	EncryptionAtRestMode encryptMode = self->db.config.encryptionAtRestMode;
-	if (encryptMode.isEncryptionEnabled()) {
+	if (encryptMode.isEncryptionEnabled() || SERVER_KNOBS->REST_KMS_ENABLE_RECRUITMENT) {
```

Comment:
nit: maybe cleaner to define a `bool` at the start of this method and assign the value as `encryptMode.isEncryptionEnabled() || SERVER_KNOBS->REST_KMS_ENABLE_RECRUITMENT` ?